tests/FSharp.Test.Utilities/TestFramework.fs

```diff
@@ -236,7 +236,7 @@ let config configurationName envVars =
     let ILASM = requirePackage (("runtime." + operatingSystem + "-" + architectureMoniker + ".Microsoft.NETCore.ILAsm") ++ coreClrRuntimePackageVersion ++ "runtimes" ++ (operatingSystem + "-" + architectureMoniker) ++ "native" ++ ILASM_EXE)
     let CORECLR_DLL = if operatingSystem = "win" then "coreclr.dll" elif operatingSystem = "osx" then "libcoreclr.dylib" else "libcoreclr.so"
     let coreclrdll = requirePackage (("runtime." + operatingSystem + "-" + architectureMoniker + ".Microsoft.NETCore.Runtime.CoreCLR") ++ coreClrRuntimePackageVersion ++ "runtimes" ++ (operatingSystem + "-" + architectureMoniker) ++ "native" ++ CORECLR_DLL)
-    let PEVERIFY_EXE = if operatingSystem = "win" then "PEVerify.exe" else "PEVerify"
+    let PEVERIFY_EXE = if operatingSystem = "win" then "PEVerify.exe" elif operatingSystem = "osx" then "PEVerify.dll" else "PEVerify"
```

Comment:
Fix running tests on macOS.

Reply:
Do tests run on linux? it feels like a similar change would need to be made for linux here.

Reply:
@baronfel 
The ComponentTests do run on Linux, I will have to check the UnitTest ones, but I'm pretty sure they do run as well (in build.sh at least).

I also need to check what gets generated when we compile PEVerify on Linux. 
But there's a separate issue (https://github.com/dotnet/fsharp/issues/9828) for proper support of running PEVerify (either via `dotnet` or factor it out as a library).